install.sh

```diff
@@ -60,6 +60,12 @@ if [[ "$FEDORA_MAJOR_VERSION" -ne "41" ]]; then
         libvdpau
 fi
 
+# Disable DKMS support in gnome-software
+if [[ "$FEDORA_MAJOR_VERSION" -ge "41" && "$IMAGE_NAME" == "silverblue" ]]; then
+    rpm-ostree override replace \
+        https://download.copr.fedorainfracloud.org/results/ublue-os/staging/fedora-41-x86_64/08203620-gnome-software/gnome-software-47.1-100.ublue.fc41.x86_64.rpm
```

Comment:
This is a brittle way to override, no?

Shouldn't we override by override replace from the ublue-os-stabing COPR?

Reply:
This was a test and it failed, you can ignore it. We need a way to get around it crashing while trying to override.

Reply:
I think I see that the problem was doing it with a replace that pulls from the repo ends up pulling in a bunch of devel packages, no?

That seems like it could be related to rpm-ostree always pulling recommended packages. I'm not sure.

Well, we can probably merge as is, as fairly soon, I expect we'll be doing an overhaul of main builds to use dnf, and I hope this will be easier to address at that time.

Reply:
This fails to build as is, so it needs some kind of fix. Marked as draft

Reply:
I'll test.

Reply:
It works now.
It's interesting seeing how `gnome-software-rpm-ostree` is getting removed, then getting installed with Ublue's version, then gets removed again lol

Reply:
Yes, thank you for the tip, i'm going to see if we can use the repo override now, rather than the direct URL to package.

Yeah, not the only place we remove stuff to have it re-added and removed again.

Reply:
Fixed!